libcontainer/integration/exec_test.go

```diff
@@ -535,14 +535,15 @@ func testPids(t *testing.T, systemd bool) {
 	}
 
 	config := newTemplateConfig(t, &tParam{systemd: systemd})
-	config.Cgroups.Resources.PidsLimit = -1
+	l := int64(-1)
+	config.Cgroups.Resources.PidsLimit = &l
```

Comment:
Why not `*config.Cgroups.Resources.PidsLimit = -1`?

Reply:
the pointer isn't initialized. I could initialize in newTemplateConfig, but that would have the same amount of code and be needless for the other tests that don't do it

Reply:
Ah, it's true, sorry.